### DISCUSSION TITLE
Why should I use an email forwarding service?

### DISCUSSION THREAD

**Discussion by ravinggenius** [forwardemail/forwardemail.net#200] [General]
I'm building a service that needs to send occasional emails (email verification and forgot password) from a single system address. I'm setting up NodeMailer, and I noticed the website mentions the Forward Email service. My domain is registered with Gandi, which gives me SMTP credentials already. I was planning to use Gandi's SMTP service directly until I saw Forward Email. Can someone please help me understand why I should use such an email forwarding service instead of my registrar? It just seems like more steps to do the same thing.

(I have gone over the FAQ. I understand Forward Email does a bunch of spam protection that my registrar may not. However my webapp is the only entity that should be sending emails, and I'm not going to be spamming anyone. What is the benefit to me?)

**Comment by titanism** [comment]
Hi there @ravinggenius.  The team here from Forward Email chiming in with a few reasons why you should use us vs any other:

1. We do not store logs (with exception of errors and outbound SMTP for ~30 days)
2. We are the _only_ provider that is 100% open-source (others claim to be open source, but if you actually look closely you will see that they only open-source certain things, such as their front-end or apps, instead of their actual back-ends; which are responsible for handling your data).
3. We are extremely reliable and performant - see our status page at https://status.forwardemail.net
4. You will get an actual human and an actual engineer to give you support (and technical support).
5. The price is only $3/mo for unlimited domains/emails/aliases (you can't find a better price)

If you have any other questions just reach out here on GitHub or at support@forwardemail.net for a more reliable support process.

**Comment by ravinggenius** [comment]
Thanks for responding. I understand those points from reading the website, but they don't really answer my question. My understanding is that I'm going to have to connect Forward Email to Gandi to actually use it. In that case, what is the advantage to connecting to Forward Mail versus directly to Gandi? It seems like "check your email in Gandi, but with extra steps". I'm assuming there's more to your service than that. That's what I'm trying to understand.

**Reply by titanism** [reply]
@ravinggenius Gandi is a domain name registrar - but they also provide some basic email-related services (e.g. forwarding).  Albeit email is not their focus, so they don't offer everything.  We are an email service provider and that is our focus.  Consider us an alternative to connecting Gandi with Gmail Business, or Gandi with Proton Mail, or Gandi with Fastmail, or Gandi with `<insert other email service here>`.

**Reply by ravinggenius** [reply]
Got it! Perfect, that's what I was looking for! Thank you.
